test/framework/easyconfigs/test_ecs/g/gzip/gzip-1.6-GCC-4.9.2.eb

```diff
@@ -13,7 +13,8 @@
 easyblock = 'ConfigureMake'
 
 name = 'gzip'
-version = '1.6'  # FIXME bug: quotes are required here to make sure this is parsed a a string, not a floating point value
+# FIXME bug: quotes are required here to make sure this is parsed a a string, not a floating point value
```

Comment:
@wpoely86 please drop this comment, it doesn't make any sense here (it's copy-pasted from the equivalent `.yeb` easyconfig)